src/besselj.jl

```diff
@@ -114,34 +217,29 @@ function besselj1(x::Float64)
     x = abs(x)
     isinf(x) && return zero(x)
 
-    if x <= 5.0
-        z = x * x
-        w = evalpoly(z, RP_j1(T)) / evalpoly(z, RQ_j1(T))
-        w = w * x * (z - 1.46819706421238932572e1) * (z - 4.92184563216946036703e1)
-        return w
-    elseif x < 25.0
-        w = 5.0 / x
-        z = w * w
-        p = evalpoly(z, PP_j1(T)) / evalpoly(z, PQ_j1(T))
-        q = evalpoly(z, QP_j1(T)) / evalpoly(z, QQ_j1(T))
-        xn = x - THPIO4(T)
-        sc = sincos(xn)
-        p = p * sc[2] - w * q * sc[1]
-        return p * SQ2OPI(T) / sqrt(x)
+    if x <= 26.0
+        x <= pi/2 && return x*evalpoly(x*x, J1_POLY_PIO2(T))
```

Comment:
Also, do we want to set a code style for this repo? I think in general I've been trying to use blue but sometimes I obsess over spaces too much 😅

Reply:
I usually _try_ to follow https://github.com/jrevels/YASGuide. I can make some spacing changes if that's ok.

Otherwise I think this is ready to merge?

Reply:
sounds good. (make any changes you want).